lib/ts/axios.ts

```diff
@@ -190,6 +191,9 @@ export function responseInterceptor(axiosInstance: any) {
                     true
                 );
```

Comment:
Add tests

Reply:
I'll add tests in a separate PR with both #136 and this merged.